apps/console/src/features/groups/components/wizard/group-basics.tsx

```diff
@@ -158,7 +158,7 @@ export const GroupBasics: FunctionComponent<GroupBasicProps> = (props: GroupBasi
 
         setUserStore(storeOptions[ 0 ].value as string);
 
-        if (isRootOrganization) {
+        if (isSuperOrganization() || isFirstLevelOrganization()) {
```

Comment:
What's the reason not to use `isRootOrganization`?

Reply:
Thought to remove it because Introducing a variable with the name `isRootOrganization` could possibly reintroduce the ambiguity between the super tenant and the first-level org that we wanted to avoid.